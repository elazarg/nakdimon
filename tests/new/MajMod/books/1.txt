חוֹקְרִים רַבִּים לֹא הִשְׁתַּכְנְעוּ ונותרו מבולבלים. לְאַחַר שמחקרו הָרִאשׁוֹן פּוּרְסַם, הִתְקַשְׁרָה אֶל שטרן אִישָּׁה שֶׁבַּעֲלָהּ, חָתָן פְּרַס נוֹבֵּל, סָבַל ממקרה חָמוּר שֶׁל אלצהיימר. "הִיא אָמְרָה, 'עַל מָה אַתָּה מְדַבֵּר, לַעֲזָאזֵל!'" אָמַר שטרן. אף-על-פי שֶׁאִישׁ לֹא טָעַן - אוֹ אוֹמֵר כַּיּוֹם - שהשכלה הִיא עֲרוּבָּה לַהֲגָנָה מִפְּנֵי שיטיון, הִתְקַשּׁוּ לְקַבֵּל אֶת הָרַעְיוֹן שֶׁמַּשֶּׁהוּ אמורפי כְּמוֹ הַשְׂכָּלָה יָכוֹל לְהָגֵן עַל הַמּוֹחַ מִפְּנֵי פְּגִיעָתָהּ הַפִיזִית הממשית שֶׁל מַחֲלָה קָשֶׁה. "פָּשׁוּט לֹא חָשַׁבְנוּ שהמוח עוֹבֵד כָּכָה," אוֹמֵר שטרן. אַךְ בְּמִידָּה הוֹלֶכֶת וְגוֹבֶרֶת נִרְאֶה שֶׁאָכֵן, כָּךְ הוּא עוֹבֵד. מֶחְקָרִים מהימנים נוֹסָפִים, כְּדוּגְמַת מֶחְקָר "המסדרים הַדָּתִיִּים" שֶׁל רֹאשׁ (עוֹד נזירות, וְגַם כְּמָרִים) בְּ-2004, חָשְׂפוּ גַּם שברמת חוּמְרָה נְתוּנָה, נִמְצְאוּ אֵצֶל המשכילים יוֹתֵר גושי סִיבִים ומשקעים - עוֹד הוֹכָחָה, אוּלַי, שבעלי הַהַשְׂכָּלָה הַרְבֵּה יוֹתֵר הָיוּ מוּגַנִּים לְמֶשֶׁךְ זְמַן רַב יוֹתֵר מִפְּנֵי הַשְׁפָּעָתָהּ הַקָּשָׁה בְּיוֹתֵר שֶׁל הַמַּחֲלָה. אוֹתָהּ קְבוּצָה גַּם מָצְאָה שֶׁלֹּא רַק רָמַת הַהַשְׂכָּלָה הָיְתָה קְשׁוּרָה לסכנת השיטיון, אֶלָּא גַּם "פְּעִילוּיוֹת מעוררות-קוגניציה". בַּמֶּחְקָר מכונן, שֶׁנָּתַן דְּחִיפָה לְתַעֲשִׂיַּית התשבצים, מָצְאָה קְבוּצַת רֹאשׁ שבמהלך תְּקוּפָה שֶׁל חָמֵשׁ שָׁנִים הָיָה לְמִי שֶׁהִרְבּוּ לְהַפְעִיל אֶת תָּאֵי הָעֶצֶב שֶׁלָּהֶם סִיכּוּי נָמוּךְ בַּחֲצִי לְפַתֵּחַ אלצהיימר. פְּעִילוּיוֹת מעוררות הוּגְדְּרוּ ככאלה, שֶׁבָּהֶן "חִיפּוּשׂ אוֹ עִיבּוּד מֵידָע" הָיָה חֵלֶק מֶרְכָּזִי, ושניתן לעשותן לְבַד (כְּדֵי לְנַטְרֵל אֶת הַשְׁפָּעַת הַחֲבָרוֹת, שתרומתה לַמּוֹחַ עֲשׂוּיָה לִהְיוֹת אַדִּירָה). כְּלוֹמַר: מִשְׂחָק ברידג' הָיָה מִחוּץ לִתְחוּם, אַךְ קְרִיאַת מגזינים ועיתונים, בִּיקּוּר בספריה, השתעשעות בְּמִשְׂחֲקֵי מִילִּים, הִשְׁתַּתְּפוּת בְּשִׁיעוּרֵי מוּזִיקָה אוֹ לְמִידַּת שָׂפָה זָרָה - כּוּלָּם נֶחְשְׁבוּ.