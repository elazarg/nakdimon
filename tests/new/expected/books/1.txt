חוֹקְרִים רַבִּים לֹא הִשְׁתַּכְנְעוּ וְנוֹתְרוּ מְבוּלְבָּלִים.
לְאַחַר שֶׁמֶּחְקָרוֹ הָרִאשׁוֹן פּוּרְסַם, הִתְקַשְּׁרָה אֶל שְׁטֵרְן אִישָּׁה שֶׁבַּעֲלָהּ, חֲתַן פְּרַס נוֹבֵּל, סָבַל מִמִּקְרֶה חָמוּר שֶׁל אַלְצְהַיְימֶר.
"הִיא אָמְרָה, 'עַל מָה אַתָּה מְדַבֵּר, לַעֲזָאזֵל!'" אָמַר שְׁטֵרְן.
אַף-עַל-פִּי שֶׁאִישׁ לֹא טָעַן - אוֹ אוֹמֵר כַּיּוֹם - שֶׁהַשְׂכָּלָה הִיא עֲרוּבָּה לַהֲגָנָה מִפְּנֵי שִׁיטָּיוֹן, הִתְקַשּׁוּ לְקַבֵּל אֶת הָרַעְיוֹן שֶׁמַּשֶּׁהוּ אָמוֹרְפִי כְּמוֹ הַשְׂכָּלָה יָכוֹל לְהָגֵן עַל הַמּוֹחַ מִפְּנֵי פְּגִיעָתָהּ הַפִיזִית הַמַּמָּשִׁית שֶׁל מַחֲלָה קָשָׁה.
"פָּשׁוּט לֹא חָשַׁבְנוּ שֶׁהַמּוֹחַ עוֹבֵד כָּכָה," אוֹמֵר שְׁטֵרְן.
אַךְ בְּמִידָּה הוֹלֶכֶת וְגוֹבֶרֶת נִרְאֶה שֶׁאָכֵן, כָּךְ הוּא עוֹבֵד.
מֶחְקָרִים מְהֵימָנִים נוֹסָפִים, כְּדוּגְמַת מֶחְקָר "הַמִּסְדָּרִים הַדָּתִיִּים" שֶׁל רַאשׁ (עוֹד נְזִירוֹת, וְגַם כְּמָרִים) בְּ-2004, חָשְׂפוּ גַּם שֶׁבְּרָמַת חוּמְרָה נְתוּנָה, נִמְצְאוּ אֵצֶל הַמַּשְׂכִּילִים יוֹתֵר גּוּשֵׁי סִיבִים וּמִשְׁקָעִים - עוֹד הוֹכָחָה, אוּלַי, שֶׁבַּעֲלֵי הַהַשְׂכָּלָה הַרְבֵּה יוֹתֵר הָיוּ מוּגַנִּים לְמֶשֶׁךְ זְמַן רַב יוֹתֵר מִפְּנֵי הַשְׁפָּעָתָהּ הַקָּשָׁה בְּיוֹתֵר שֶׁל הַמַּחֲלָה.
אוֹתָהּ קְבוּצָה גַּם מָצְאָה שֶׁלֹּא רַק רָמַת הַהַשְׂכָּלָה הָיְתָה קְשׁוּרָה לְסַכָּנַת הַשִּׁיטָּיוֹן, אֶלָּא גַּם "פְּעִילוּיוֹת מְעוֹרְרוֹת-קוֹגְנִיצְיָה".
בְּמֶחְקָר מְכוֹנֵן, שֶׁנָּתַן דְּחִיפָה לְתַעֲשִׂיַּית הַתַּשְׁבְּצִים, מָצְאָה קְבוּצַת רַאשׁ שֶׁבְּמַהֲלַךְ תְּקוּפָה שֶׁל חָמֵשׁ שָׁנִים הָיָה לְמִי שֶׁהִרְבּוּ לְהַפְעִיל אֶת תָּאֵי הָעֶצֶב שֶׁלָּהֶם סִיכּוּי נָמוּךְ בְּחֵצִי לְפַתֵּחַ אַלְצְהַיְימֶר.
פְּעִילוּיוֹת מְעוֹרְרוֹת הוּגְדְּרוּ כְּכָאֵלֶּה, שֶׁבָּהֶן "חִיפּוּשׂ אוֹ עִיבּוּד מֵידָע" הָיָה חֵלֶק מֶרְכָּזִי, וְשֶׁנִּיתָּן לַעֲשׂוֹתָן לְבַד (כְּדֵי לְנַטְרֵל אֶת הַשְׁפָּעַת הַחֲבֵרוּת, שֶׁתְּרוּמָתָהּ לַמּוֹחַ עֲשׂוּיָה לִהְיוֹת אַדִּירָה).
כְּלוֹמַר: מִשְׂחַק בְּרִידְג' הָיָה מִחוּץ לַתְּחוּם, אַךְ קְרִיאַת מָגָזִינִים וְעִיתּוֹנִים, בִּיקּוּר בַּסִּפְרִיָּה, הִשְׁתַּעְשְׁעוּת בְּמִשְׂחֲקֵי מִילִּים, הִשְׁתַּתְּפוּת בְּשִׁיעוּרֵי מוּזִיקָה אוֹ לְמִידַּת שָׂפָה זָרָה - כּוּלָּם נֶחְשְׁבוּ.